bundle/src/main/java/com/adobe/acs/commons/properties/util/PropertyAggregatorUtil.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2020 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.adobe.acs.commons.properties.util;
+
+import com.adobe.acs.commons.properties.PropertyConfigService;
+import com.day.cq.wcm.api.Page;
+import org.apache.sling.api.resource.ValueMap;
+
+import java.util.Map;
+import java.util.Set;
+
+public class PropertyAggregatorUtil {
+
+    private PropertyAggregatorUtil() {
+        throw new IllegalStateException("PropertyAggregatorUtil is a utility class, it should not be instantiated.");
+    }
+
+    /**
+     * Add the properties of a page to the given map.  Excluded properties are found in the
+     * {@link PropertyConfigService} service.
+     *
+     * @param map                   the map that should be updated with the properties and their values
+     * @param page                  the page containing properties
+     * @param prefix                the prefix to apply to the
+     * @param propertyConfigService the {@link PropertyConfigService} used to check type and exclusion
+     */
+    public static void addPagePropertiesToMap(Map<String, Object> map, Page page, String prefix,
```

Comment:
Will this ever be reused, or should it be placed into `AllPagePropertiesContentVariableProvider`?

Reply:
I kept it outside in the Util class because ideally this is a method developers use so that both the type limitations and property exclusion are enforced and the same scheme of 'prefix.key' is enforced.

Reply:
I think that definitely covers why we have `addPropertiesToMap()` in the util, just wondering if `addPagePropertiesToMap` wrapper should be here since it seems specific to the impl class

Reply:
This is true - it made sense to be here when the universal source of properties was a page, but since it's updated to be a request I'll move this